pkg/apis/flowcontrol/fuzzer/fuzzer.go

```diff
@@ -21,27 +21,31 @@ import (
 
 	runtimeserializer "k8s.io/apimachinery/pkg/runtime/serializer"
 	"k8s.io/kubernetes/pkg/apis/flowcontrol"
+	"k8s.io/utils/ptr"
 )
 
 // Funcs returns the fuzzer functions for the flowcontrol api group.
 var Funcs = func(codecs runtimeserializer.CodecFactory) []interface{} {
 	return []interface{}{
 		func(obj *flowcontrol.LimitedPriorityLevelConfiguration, c fuzz.Continue) {
 			c.FuzzNoCustom(obj) // fuzz self without calling this function again
+
+			// NOTE: setting a zero value here will cause the roundtrip
+			// test (from internal to v1beta2, v1beta1) will fail
```

Comment:
typo here: it reads "will cause ... will fail"; I think you want "will cause ... to fail".